src/components/menu-top/LoadModal.vue

```diff
@@ -81,8 +81,10 @@ export default Vue.extend({
       reader.onload = (): void => {
         if (reader.result && reader.result instanceof ArrayBuffer) {
           try {
+            const title = this.file ? this.file.name : "";
```

Comment:
It'd look a bit cleaner if we remove the ".shw" from the title

Reply:
Agreed, I'll parse it out.